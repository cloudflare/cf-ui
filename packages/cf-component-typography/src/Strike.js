const React = require('react');
const { PropTypes } = React;

class Strike extends React.Component {
  render() {
    return (
      <s className="cf-strike">
        {this.props.children}
      </s>
    );
  }
}

Strike.propTypes = {
  children: PropTypes.node
};

module.exports = Strike;
