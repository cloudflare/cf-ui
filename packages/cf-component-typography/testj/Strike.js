import React from 'react';
import renderer from 'react-test-renderer';
import Strike from '../src/Strike';

test('should render', () => {
  const component = renderer.create(<Strike>Strike</Strike>);
  expect(component.toJSON()).toMatchSnapshot();
});
