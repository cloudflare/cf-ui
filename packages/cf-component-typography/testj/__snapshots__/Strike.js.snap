// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Strike should render 1`] = `
<s
  className="cf-strike"
>
  Strike
</s>
`;
